### ISSUE TITLE
Can't compile under windows using clang 

### ISSUE THREAD

**Issue by markusstephanides** [pr0g/sdl-bgfx-imgui-starter#18]
Hi!

The starter works perfectly well on windows with MSVC but won't compile for me if I use clang.

My setup:

1. Install msys2: https://www.msys2.org/
2. Open MSYS UCRT64
3. pacman -S mingw-w64-x86_64-cmake
4. pacman -S mingw-w64-x86_64-ninja
5. pacman -S mingw-w64-x86_64-clang

And I configured the project using configure-ninja.bat.
Then I started building and it seems like it doesn't know the cstdint types:

```
bgfxproject/third-party/build/src/bgfx/bgfx/3rdparty/glslang/glslang/MachineIndependent/SymbolTable.h:662:22: note: 'uint64_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?  

bgfxproject/third-party/build/src/bgfx/bgfx/3rdparty/glslang/glslang/MachineIndependent/SymbolTable.h:663:18: error: 'uint32_t' does not name a type
  663 |     static const uint32_t MaxLevelInUniqueID = 127;
      |                  ^~~~~~~~  

bgfxproject/third-party/build/src/bgfx/bgfx/3rdparty/glslang/glslang/MachineIndependent/SymbolTable.h:663:18: note: 'uint32_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
```

Do you have an idea on how to solve this?

**Comment by markusstephanides** [comment]
Update: I think it was fixed by updating glslang which is already integrated into the bgfx repo, so I guess it just needs an update.

**Comment by pr0g** [comment]
Hi @markusstephanides,

Oh, interesting! Does it look like I need to update the dependency on bgfx then? (if they've fixed this issue upstream?) I should be able to do this later this week (I might see if I can update the other dependencies as well).

Thanks for letting me know and glad you're unblocked in the meantime 👍

**Comment by pr0g** [comment]
I've just pushed a branch with all dependencies updated - https://github.com/pr0g/sdl-bgfx-imgui-starter/compare/main...update-dependencies

I did a quick test on macOS and all seems fine but if you could verify it fixes the problem you're seeing that would be great! If it looks good I'll merge tomorrow 👍

**Comment by markusstephanides** [comment]
Looking good, thank you very much!

**Comment by hultonha** [comment]
> Looking good, thank you very much!

Great to hear! Thanks for letting me know, I'll get this merged this evening 👍

**Comment by pr0g** [comment]
> Looking good, thank you very much!

Great to hear! Thank you for letting me know. I'll get this merged this evening 👍

**Comment by pr0g** [comment]
The branch is now merged (see f57f8cd600061b4b5d4138a5054b4cbc8d0209ec). Closing this issue. If you run into any other problems give me a shout, thanks!
